src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -441,12 +441,12 @@ private DbContext TryCreateContextUsingAppCode(Type dbContextType, Type startupT
                 // Use EF design APIs to get the DBContext instance.
                 var operationHandler = new OperationReportHandler();
                 var operationReporter = new OperationReporter(operationHandler);
+                // EF infers the environment (Development/ Production) based on the environment variable
+                // ASPNETCORE_ENVIRONMENT. This should already be set up by the CodeGeneration.Design process.
                 var dbContextOperations = new DbContextOperations(
                     operationReporter,
                     dbContextType.GetTypeInfo().Assembly,
-                    startupType.GetTypeInfo().Assembly,
-                    "Development",
-                    Path.GetDirectoryName(_projectContext.ProjectFullPath));
```

Comment:
@mlorbetske, the startupType's assembly is still needed. 
The `environment` and `contentRootPath` parameters are no longer needed. 
See https://github.com/aspnet/EntityFramework/issues/7050#issuecomment-297833793 for reference.